docs/release-notes/NuGet-5.0-RTM.md

```diff
@@ -0,0 +1,156 @@
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload 
+
+<sup>2</sup>Available as an optional install with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.0
+
+* Support solution filter when restoring the solution via CLI or in Visual Studio - [#5820](https://github.com/NuGet/Home/issues/5820)
+* `BuildTransitive` folder enables packages to transitively contribute targets/props to the host project - [#6091](https://github.com/NuGet/Home/issues/6091)
+* Better support for PackageReference scenarios in NuGet IVs APIs - [#7005](https://github.com/NuGet/Home/issues/7005), [#7493](https://github.com/NuGet/Home/issues/7493)
+* `dotnet pack project.json` has been deprecated - [#7928](https://github.com/NuGet/Home/issues/7928)
+* Gen 1 Credential Provider plugin is no longer supported - [#7819](https://github.com/NuGet/Home/issues/7819)
+
+## Issues fixed in this release
+
+**Bugs**
+
+* When doing a NoOp restore, avoid obj\*dgspec.json write - [#7854](https://github.com/NuGet/Home/issues/7854)
+
+* Permissions on files created inside ~/.nuget are too open - [#7673](https://github.com/NuGet/Home/issues/7673)
+
+* dotnet list package --outdated doesn't work with sources that need auth - [#7605](https://github.com/NuGet/Home/issues/7605)
+
+* NuGet.VisualStudio.IVsPackageInstaller - calling on a project with no package references always uses packages.config, even if the default is set to PackageReference - [#7005](https://github.com/NuGet/Home/issues/7005)
+
+* PMC: Update-Package reinstall fails ("Unable to find package") on delisted packages. - [#7268](https://github.com/NuGet/Home/issues/7268)
+
+* Add third party notice in our repo and VSIX - [#7409](https://github.com/NuGet/Home/issues/7409)
+
+* NuGet.VisualStudio.IVsPackageInstaller.InstallPackage should install latest version when no version given - [#7493](https://github.com/NuGet/Home/issues/7493)
+
+* --interactive support for dotnet nuget push - [#7519](https://github.com/NuGet/Home/issues/7519)
+
+* When restoring with lock file, NU1603 warning shouldn't be raised. - [#7529](https://github.com/NuGet/Home/issues/7529)
+
+* NuGet should not print project path during restore with minimal logging - [#7647](https://github.com/NuGet/Home/issues/7647)
+
+* --interactive support for dotnet remove package - [#7727](https://github.com/NuGet/Home/issues/7727)
+
+* Add back NuGet.Packaging.Core with TypeForwardedTo attrs - [#7768](https://github.com/NuGet/Home/issues/7768)
+
+* plugins_cache needs shorter path to work well - [#7770](https://github.com/NuGet/Home/issues/7770)
+
+* Prefer path for msbuild discovery if user didn't ask for specific msbuild version - [#7786](https://github.com/NuGet/Home/issues/7786)
+
+* nuget.exe /? should list correct msbuild versions - [#7794](https://github.com/NuGet/Home/issues/7794)
```

Comment:
```suggestion
* `nuget.exe /?` should list correct msbuild versions - [#7794](https://github.com/NuGet/Home/issues/7794)
```

Reply:
i've edited the title of issue to have tick marks -- (we'll try to get in the habit of that, as these release notes are auto generated from the titles).

@karann-msft  - please manually add the tick marks to this note.